.github/workflows/go.yaml

```diff
@@ -9,57 +9,56 @@ jobs:
         os: [ubuntu-latest, macOS-latest]
     runs-on: ${{matrix.os}}
     steps:
-    - name: Checkout code
-      uses: actions/checkout@v1
-      with:
-        path: src/github.com/nats-io/nats-surveyor
-    - name: Setup Go
-      uses: actions/setup-go@v1
-      with:
-        go-version: ${{matrix.go}}
-    - name: Install deps
-      shell: bash --noprofile --norc -x -eo pipefail {0}
-      env:
-        GO111MODULE: 'on'
-      run: |
-        export GOPATH="$RUNNER_WORKSPACE"
-        go get -v github.com/wadey/gocovmerge
-        go get -v github.com/golangci/golangci-lint/cmd/golangci-lint
-    - name: Lint
-      shell: bash --noprofile --norc -x -eo pipefail {0}
-      env:
-        GO111MODULE: 'on'
-      run: |
-        export GOPATH="$RUNNER_WORKSPACE"
+      - name: Checkout code
```

Comment:
I don't see what the changes here are other than whitespace added.  Was this intentional?

Reply:
I removed `--enable gochecknoinits` as init() is pretty much how prom works in all their examples so sticking with what others might expect.

And then because I use redhats YAML plugin that reformats things to their standard we get a load of noise for free, can undo all the format edits

Reply:
No worries, just curious.